examples/resources/sentry_metric_alert/resource.tf

```diff
@@ -20,6 +20,18 @@ resource "sentry_metric_alert" "main" {
     threshold_type  = 0
   }
 
+  trigger {
```

Comment:
Perhaps it doesn't make much sense for us to provide a "demo" file that won't load (since 99999 doesn't correspond with an existing Sentry integration).